main/ZactuatorSomfy.ino

```diff
@@ -71,7 +71,9 @@ void MQTTtoSomfy(char* topicOri, JsonObject& jsonData) {
     SomfyRemote somfyRemote(RF_EMITTER_GPIO, somfyRemotes[remoteIndex], &rollingCodeStorage);
     somfyRemote.sendCommand(command);
 #    ifdef ZradioCC1101
-    ELECHOUSE_cc1101.SetRx(CC1101_FREQUENCY); // set Receive on
+    if (CC1101_FREQUENCY_SOMFY != receiveMhz)
+      ELECHOUSE_cc1101.Init();
```

Comment:
why is `Init()` called here?

Reply:
During testing I found that the frequency was not taking effect, and initialing the transceiver resolved the issue.  Reviewed the CC1101 data sheets and examples and did not see this scenario covered.  So it is a bit of hack.

Reply:
I'm using the Somfy module and it sends with 433.42 MHz and then switches back to 433.92 MHz to receive PiLight signals. And it works without calling `ELECHOUSE_cc1101.Init();`.

Reply:
During my testing I was jumping between 433 MHz and 303 MHz, and I found that when just doing a SetRx() that it was not picking up transmissions.  After adding the Init() it started working.  Am thinking that small frequency jumps may be okay ( your experience ) but large jumps may not be.  But without knowing a valid range, its a crap shot.

Reply:
Ok, `Init()` fixed the symptoms, but I don't think this is the best solution to the problem. Calling Init restarts the SPI connection and the CC1101 module https://github.com/LSatan/SmartRC-CC1101-Driver-Lib/blob/master/ELECHOUSE_CC1101_SRC_DRV.cpp#L141

Reply:
After reviewing the data sheet further and looking into why having different transmit and receive frequencies worked, deduced that the receiver needed to be idled when changing receive frequencies.  This line here [here](https://github.com/LSatan/SmartRC-CC1101-Driver-Lib/blob/f860157b17b2449768d18ca46c6cb5744bad1971/ELECHOUSE_CC1101_SRC_DRV.cpp#L975)

After a few hours of testing, this resolves it, and removes the kludge approach

Reply:
PS Raised this a question against https://github.com/LSatan/SmartRC-CC1101-Driver-Lib/issues/65

Reply:
Nice, thanks for looking into this and raising the issue in the library project. We can merge it with this workaround but should keep in mind to update to the new library version when available.